cmd/crates/soroban-spec-typescript/src/lib.rs

```diff
@@ -293,35 +282,29 @@ pub fn entry_to_method_type(entry: &Entry) -> String {
 ",
             )
         }
-        Entry::ErrorEnum { doc, cases, .. } => {
+        Entry::ErrorEnum { doc, cases, name } => {
             let doc = doc_to_ts_doc(doc, None, 0);
-            let cases = cases
-                .iter()
-                .enumerate()
-                .map(|(i, c)| {
-                    if c.doc.is_empty() {
-                        format!(
-                            "{}  {}: {{message:\"{}\"}}",
-                            if i == 0 { "" } else { "\n" },
-                            c.value,
-                            c.name
-                        )
-                    } else {
-                        format!(
-                            "{}{}  {}: {{message:\"{}\"}}",
-                            if i == 0 { "" } else { "\n" },
-                            doc_to_ts_doc(&c.doc, None, 1),
-                            c.value,
-                            c.name
-                        )
-                    }
-                })
-                .join(",\n");
-            format!("{doc}export const Errors = {{\n{cases}\n}}")
+            let cases = cases.iter().map(error_case_to_ts).join(",\n");
+            let name = if name == "Error" {
+                format!("{name}s")
```

Comment:
Do you know why the s is added? I see you're just copying the behaviour from above.

Reply:
I suspect the original guess was that `Error` is a reserved word in JS. It’s not; we can name the export `Error` and compile with no problem. It does make for unexpected name collisions though, with the built in `Error`. 

I figured matching the logic above would make for a more consistent file as well as less of a change from the previous behavior.

Reply:
👍